docs/project/glossary.md

```diff
@@ -44,7 +44,7 @@ terminology.
 | Redhawk | Codename for experimental minimal managed code runtime that evolved into [CoreRT](https://github.com/dotnet/corert/). |
 | SDK | Software Development Kit. The [.NET SDK](https://docs.microsoft.com/dotnet/core/sdk) contains the .NET CLI, .NET libraries and runtime, and the dotnet driver. |
 | SEH | [Structured Exception Handling](https://docs.microsoft.com/windows/win32/debug/structured-exception-handling). Unified mechanism for handling hardware and software exceptions on Windows. |
-| SOS | [Son of Strike](https://docs.microsoft.com/archive/blogs/jasonz/sos-debugging-of-the-clr-part-1). The debugging extension for DbgEng based debuggers. Uses the DAC as an abstraction layer for its operation. |
+| SOS | [Son of Strike](https://web.archive.org/web/20180726075341/https://blogs.msdn.microsoft.com/jasonz/2003/10/21/sos-debugging-of-the-clr-part-1/). The debugging extension for DbgEng based debuggers. Uses the DAC as an abstraction layer for its operation. |
```

Comment:
We should probably just point to the docs instead:
```suggestion
| SOS | [Son of Strike](https://learn.microsoft.com/en-us/dotnet/framework/tools/sos-dll-sos-debugging-extension). The debugging extension for DbgEng based debuggers. Uses the DAC as an abstraction layer for its operation. |
```

Reply:
@stephentoub I don't see "Son of Strike" mentioned anywhere at https://learn.microsoft.com/dotnet/framework/tools/sos-dll-sos-debugging-extension. That doesn't work to explain the origin of the term.

Reply:
Does it need to explain the origin, as opposed to what it is?

Reply:
@stephentoub AFAIR that was my reason for linking to Jason Zander's blog post in the [original pr](https://github.com/dotnet/coreclr/pull/134). But I agree that a link to the documentation is probably more useful to most people reading the glossary. It's just a bit odd name without the history behind - maybe we could add it as a footnote or something?

(Also I think the reason the post has disappeared in the most recent blog migration is because it was posted as "Anonymous" which was probably just a mistake)